miniapps/navier/navier_solver.cpp

```diff
@@ -113,7 +113,8 @@ void NavierSolver::Setup(double dt)
 
    sw_setup.Start();
 
-   pmesh_lor = new ParMesh(pmesh, order, BasisType::GaussLobatto);
+   pmesh_lor = new ParMesh(
+      ParMesh::MakeRefined(*pmesh, order, BasisType::GaussLobatto));
```

Comment:
Another place where `new ParMesh(ParMesh::MakeRefined(...))` is used.

Reply:
Yeah, I believe this is the only such occurrence in the repo now. Personally I don't think it's the end of the world, and would prefer to leave it as is rather than restructure this part of `navier_solver`.

Reply:
@jandrej, what do you think? Does this type of construction make sense to you? I find it strange and not very natural for new users. However, this is deeper in the `navier` miniapp, so it may be fine.

Reply:
It's ok with me